src/Symfony/Component/Workflow/StateMachine.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
```

Comment:
should be added to 3.4

Reply:
see https://github.com/symfony/symfony/pull/30618